client/src/components/App.jsx

```diff
@@ -74,6 +75,7 @@ export default class App extends React.Component {
     return (
       <>
         <GlobalStyle />
+        {JSON.stringify(this.state.overview)}
```

Comment:
Is it for testing purpose to see what the obj looks like?

Reply:
ah yes (it was a good way to make sample data), but meant to take it out. thanks!